src/scale.ts

```diff
@@ -262,6 +262,11 @@ export interface Scale {
    */
   zero?: boolean;
 
+  /**
+   * The color space for scale/
+   */
+  interpolate?: 'rgb'| 'lab' | 'hcl' | 'hsl' | 'hsl-long' | 'hcl-long';
```

Comment:
What about rgb-basis?  Please look at d3 color interpolator again and make sure you include all of them.